lib/ioc-rc

```diff
@@ -543,10 +543,12 @@ __rc_jails () {
         local name="$(zfs get -H -o value org.freebsd.iocage:host_hostuuid \
                     $jail)"
         local boot="$(zfs get -H -o value org.freebsd.iocage:boot $jail)"
+        local _jail_hostid="$(zfs get -H -o value org.freebsd.iocage:hostid \
+                        $jail)"
         local priority="$(zfs get -H -o value org.freebsd.iocage:priority \
                         $jail)"
 
-        if [ "$boot" == "on" ] ; then
+        if [ "$boot" == "on" ] && [ "$_jail_hostid" == "$hostid" ] ; then
```

Comment:
I guess it depends on what you are used to! I find the first one more readable simply because I so seldom encounter the second form. heh heh

I will note the preference for any future pull reqs though. Good to know. Maybe this and any other syntax preferences could be briefly outlined in a section of the readme or a "CONTRIBUTING" doc?